common/zoom/zoom-scheduled-meeting.ts

```diff
@@ -80,7 +80,7 @@ const createZoomMeeting = async (zoomUsers: ZoomUser[], startTime: Date, isCours
                     waiting_room: isCourse ? true : false,
                     breakout_room: isCourse ? true : false,
                     recurrence: endDateTime && {
-                        end_date_time: new Date(endDateTime.setHours(24, 0, 0, 0)),
+                        end_date_time: moment(endDateTime).tz(tz).format('YYYY-MM-DDTHH:mm:ss'),
```

Comment:
What you actually did was fixing the passing by reference of a date which that gets maninpulated here and manipulates the last date before creating the appointment. But good job! 👍🏻